tests/conftest.py

```diff
@@ -142,6 +142,55 @@ def sample_future_events(mocker):
     )
     return events
 
+@pytest.fixture
+def sample_all_events(mocker):
+    events = [
+        {
+            "title": "Test title 5",
+            "event_type": "Talk",
+            "image_filename": "event.png",
+            "event_dates": [{
+                "event_datetime": "2018-12-30 19:00"
+            }],
+            "event_state": "approved"
+        },
+        {
+            "title": "Test title 6",
+            "event_type": "Talk",
+            "image_filename": "event.png",
+            "event_dates": [{
+                "event_datetime": "2018-12-31 19:00"
+            }],
+            "event_state": "approved"
+        },
+        {
+            "title": "Test title 7",
+            "event_type": "Introductory Course",
+            "past": "Past Event",
```

Comment:
Rather than setting `past` to `Past Event`, it would be better to set it to `True`, like it would in `index.py`, and rather than returning always all the events, there should be a distinction between future and past events. So I wouldn't mock the `ApiClient.get` but rather `get_events_past_year` and `get_events_in_future`